src/plugins/notebook/components/notebook-menu-switcher.vue

```diff
@@ -80,15 +80,15 @@ export default {
                 if (domainObject.location) {
                     const defaultPath = `${domainObject.name} - ${defaultNotebook.section.name} - ${defaultNotebook.page.name}`;
 
-                    notebookTypes.push({
+                    await notebookTypes.push({
```

Comment:
This is odd. Maybe setNotebookTypes shouldn't be async and remove the awaits?

Reply:
@nikhilmandlik Please take a look at this one. Docs: https://eslint.org/docs/rules/require-await

Reply:
Yup, lets remove async from `async setNotebookTypes() {`

Reply:
done